src/Microsoft.PowerShell.Commands.Utility/commands/utility/AddType.cs

```diff
@@ -553,149 +509,110 @@ internal string GenerateTypeSource(string typeNamespace, string name, string sou
-            }
-
             switch (language)
             {
-                case Language.VisualBasic:
+                case Language.CSharp:
                     return
-                        "Imports System\n" +
-                        "Imports System.Runtime.InteropServices\n" +
+                        "using System;\n" +
+                        "using System.Runtime.InteropServices;\n" +
                         "{0}" +
                         "\n";
-                case Language.JScript:
+                case Language.VisualBasic:
                     return
-                        "import System;\n" +
-                        "import System.Runtime.InteropServices;\n" +
+                        "Imports System\n" +
+                        "Imports System.Runtime.InteropServices\n" +
                         "{0}" +
                         "\n";
             }
+
+            Diagnostics.Assert(false, "GetUsingTemplate: Unsupported language family.");
+
             return null;
         }
 
         // Generate the code for the using statements
-        internal string GetUsingSet(Language language)
+        private string GetUsingSet(Language language)
         {
             StringBuilder usingNamespaceSet = new StringBuilder();
-            if (IsCSharp(language))
-            {
-                foreach (string namespaceValue in UsingNamespace)
-                {
-                    usingNamespaceSet.Append("using " + namespaceValue + ";\n");
-                }
-            }
-            else
-            {
-                switch (language)
-                {
-                    case Language.VisualBasic:
-                        foreach (string namespaceValue in UsingNamespace)
-                        {
-                            usingNamespaceSet.Append("Imports " + namespaceValue + "\n");
-                        }
-                        break;
 
-                    case Language.JScript:
-                        foreach (string namespaceValue in UsingNamespace)
-                        {
-                            usingNamespaceSet.Append("import " + namespaceValue + ";\n");
-                        }
-                        break;
-                }
+            switch (language)
+            {
+                case Language.CSharp:
+                    foreach (string namespaceValue in UsingNamespace)
+                    {
+                        usingNamespaceSet.Append("using " + namespaceValue + ";\n");
+                    }
+                    break;
+                case Language.VisualBasic:
+                    foreach (string namespaceValue in UsingNamespace)
+                    {
+                        usingNamespaceSet.Append("Imports " + namespaceValue + "\n");
+                    }
+                    break;
+                default:
+                    {
+                        Diagnostics.Assert(false, "GetUsingSet: Unsupported language family.");
+                    }
```

Comment:
No need to put the assert in a code block `{}`
One more minor thing -- the assert is put in `default` case here while not in the above similar switch code, maybe you can keep them consistent.

Reply:
The PR is formal cleanup to reduce the size of subsequent PRs to simplify a code review (my develop branch has over 1200 changed lines and I still haven't done all the work) - latter we'll again edit the code. So I don't think it makes sense to make today such minor changes.

Reply:
If you are going to push more commits to the same PR, then just ignore the comments now.
If you want this PR to be merged before doing the further work, then I think you need to resolve those comments.

Reply:
> One more minor thing --

The minor part is not a blocking comment, you can leave it as is.

Reply:
About `Diagnostics.Assert` - code block removed.
About minor comment - here we should return value, above we use formal `return null;`

Reply:
Thanks for the update. I removed the extra spaces before `Diagnostics.Assert`